lib/assets/javascripts/init/tinymce.js

```diff
@@ -84,11 +84,11 @@ $(document).ready(function() {
     toolbar1: "undo redo | bold italic underline strikethrough |  alignleft aligncenter alignright alignjustify | styleselect | fontsizeselect",
     toolbar2: "forecolor backcolor | bullist numlist outdent indent | link unlink anchor image media | removeformat | code preview fullscreen",
     language: $("html").attr("lang"),
-    extended_valid_elements : "div[*],span[*],iframe[src|width|height|name|align|frameborder|scrolling|id|onload],applet[code|codebase|archive|name|id|width|height|param],video[*],source[*],audio[*]",
+    extended_valid_elements: "div[*],span[*],iframe[src|width|height|name|align|frameborder|scrolling|id|onload],applet[code|codebase|archive|name|id|width|height|param],video[*],source[*],audio[*]",
     //paste_word_valid_elements: "@[style,class],-strong/b,-em/i,-span,-p,-ol,-ul,-li,-table,-tr[*],-td[colspan|rowspan],-th[*],-thead,-tfoot,-tbody,-a[href|name],sub,sup,strike,br,u",
     menubar: "edit format insert table view",
     browser_spellcheck : true,
-    //fontsize_formats: "8pt 10pt 12pt 14pt 18pt 24pt 36pt",
+    fontsize_formats: "8pt 10pt 12pt 14pt 18pt 24pt 36pt",
```

Comment:
Identifier 'fontsize_formats' is not in camel case.<br>Mixed double and single quotes.